frontend/src/app/activity-page/activity-page.component.ts

```diff
@@ -9,7 +10,9 @@ import { ViewEncapsulation } from '@angular/core';
 })
 export class ActivityPageComponent implements OnInit {
 
-  constructor() { }
+  constructor(private renderer: Renderer2) {
+    this.renderer.addClass(document.body, 'bootstrap-grid');
+   }
```

Comment:
This should not have to be added. Maje a container around the html if you need to add a class